### ISSUE TITLE
High dropped flows

### ISSUE THREAD

**Issue by ppoudrier** [ntop/ntopng#959]
Hi, I installed the dev version of ntopng and nprobe and I see a lot of dropped flows. It seem related to mysql

![image](https://cloud.githubusercontent.com/assets/25464346/22519328/ef18cb92-e87d-11e6-8413-b4372e9a3f51.png)

I tried to see if anything goes wrong with mysql but I can't see anything. The iowait is still very low (0-5%)

Also, just to let you know, the Ingress-Egress traffic at the bottom of the page is not correct : 
![image](https://cloud.githubusercontent.com/assets/25464346/22519439/40770b20-e87e-11e6-9479-dbe1adc2b1f2.png)
It's almost the same on both which is not correct.

Thank you

**Comment by simonemainardi** [comment]
you are trying to push 800+ flows per second in mysql. Make sure your mysql installation can actually keep up with the ntopng export speed. Ntopng start dropping flows when it sees that mysql is not fast enough.

also browse older solved issues:
- https://github.com/ntop/ntopng/issues/830
- https://github.com/ntop/ntopng/issues/796

**Comment by ppoudrier** [comment]
Thank you for your quick answer. I was not aware that the db engine was myiasm. I was tuning innodb. I did try to tune myiasm but unfortunately, it still drop around 6000 flows each second.
I looked at the running process in mysql and I only see 1 insert at a time, never more. 

I read somewhere that it's a good practice to use insert delayed instead of pure insert, but I don't think that's the problem right now.
